website/source/docs/job-specification/reschedule.html.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: "docs"
+page_title: "reschedule Stanza - Job Specification"
+sidebar_current: "docs-job-specification-reschedule"
+description: |-
+  The "reschedule" stanza specifies the group's rescheduling strategy upon task failures.
+  The reschedule strategy can be configured with number of attempts and a time interval.
+  Nomad will attempt to reschedule failed allocations on to another node only after
```

Comment:
Nomad will only attempt to reschedule failed allocations on to another node only after any local [restarts](docs/job-specification/restart.html) have been exceeded.